### PR TITLE
Copter 3.1.5

### PR DESCRIPTION
**Pull request by qq277250411**
#5406

### CODE REVIEW

**Comment by qq277250411** [comment]
?

**Comment by qq277250411** [comment]
Tools/scripts/install-prereqs-ubuntu.sh ??

File: APMrover2/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -571,12 +571,10 @@ bool GCS_MAVLINK_Rover::try_send_message(enum ap_message id)
         break;
 
     case MSG_MAG_CAL_PROGRESS:
-        CHECK_PAYLOAD_SIZE(MAG_CAL_PROGRESS);
```
Comment (qq277250411) [inline APMrover2/GCS_Mavlink.cpp]: CHECK_PAYLOAD_SIZE（MAG_CAL_PROGRESS）;

File: APMrover2/GCS_Mavlink.cpp [hunk 2]
```diff
@@ -571,12 +571,10 @@ bool GCS_MAVLINK_Rover::try_send_message(enum ap_message id)
...
         rover.compass.send_mag_cal_progress(chan);
         break;
 
     case MSG_MAG_CAL_REPORT:
-        CHECK_PAYLOAD_SIZE(MAG_CAL_REPORT);
```
Comment (qq277250411) [inline APMrover2/GCS_Mavlink.cpp]: -         CHECK_PAYLOAD_SIZE（MAG_CAL_REPORT）;

File: ArduCopter/config.h [hunk 3]
```diff
@@ -447,7 +447,7 @@
 #endif
 
 #ifndef ACRO_Y_EXPO_DEFAULT
- #define ACRO_Y_EXPO_DEFAULT        0.3f
```
Comment (qq277250411) [inline ArduCopter/config.h]: - ＃定义 ACRO_Y_EXPO_DEFAULT         0。3F

**Comment by rmackay9** [comment]
looks like a mistake.

**Pull request closed without merging**